src/main/java/rx/internal/util/ObjectPool.java

```diff
@@ -54,11 +49,11 @@ private ObjectPool(final int min, final int max, final long validationInterval)
         // initialize pool
         initialize(min);
 
-        schedulerWorker = Schedulers.computation().createWorker();
```

Comment:
Can we at least separate this from this PR so it can be discussed and tested independently?